### PR TITLE
[RE] feature/Claudia-4395

### PR DESCRIPTION
**Pull request by flopezag**
#### Reviewers
@jesuspg  @hmunfru  

#### Description
Rename the name property of the pom files: CLAUDIA-4395
Minor changes

#### Testing
Locally

### CODE REVIEW

**Comment by jesuspg** [comment]
package names should be in lowercase

File: core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/AsynManagerImplTest.java [hunk 1]
```diff
@@ -32,16 +32,12 @@
 import static org.mockito.Mockito.when;
 
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-
+    import java.util.List;
```
Comment (hmunfru) [inline core/src/test/java/com/telefonica/euro_iaas/paasmanager/manager/AsynManagerImplTest.java]: These spaces??

**Comment by hmunfru** [comment]
it looks ok, mainly you have change the group id and the package name. I have already assembly the project and it seems it works

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**